docs/debuggers_python.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Python Debuggers in Tilt
+layout: docs
+---
+
+Debuggers are an invaluable part of many development workflows. When you're running your app(s) in Kubernetes, using debuggers suddenly gets much harder, but Tilt's port forwarding functionality makes it easy to use any debugger that exposes a port for external connect.
```

Comment:
```suggestion
Debuggers are an invaluable part of many development workflows, and running your app(s) in Kubernetes makes using debuggers seemingly harder. But if your debugger works remotely, Tilt's port forwarding makes it easy to connect to it.
```

Reply:
reworked this paragraph a bit before i saw this comment, lmk what you think

Reply:
SGTM!